src/js/gui.js

```diff
@@ -59,7 +60,6 @@ class GuiControl {
             'osd',
             'servos',
             'transponder',
```

Comment:
similarly, `defaultCloudBuildTabOptions` should be deleted for the same reasons as per my comment here: https://github.com/betaflight/betaflight-configurator/pull/3512#issuecomment-1635789348

Reply:
Disagree. Without cloudbuild `defaultCloudBuildTabOptions` are concatenated into `defaultAllowedTabs.`

Reply:
1/2) yes I understand that when not present the logic won't apply, but that doesn't address the fundamental issue in that the cloud build system is not the source of truth, as you can build firmware locally and connect to it using the configurator. The firmware should be used to check for feature availability in all cases, both cloud build and local build.

what we need is a solution that does not need any cloud build artifacts; Cloud build api use should be for building/flashing of firmware only, and not for configurator interaction with the FC firmware.

Reply:
The idea behind this addition is to hide certain features with cloud build in mind.
Until now there are no issues reported except the virtual VTX feature which is addressed in the PR now.

> what we need is a solution that does not need any cloud build artifacts;

Yes would be nice to come up with a solution in firmware itself. Which extends to configuration of certain items like receiver etc.

Also we use cloud build options to preset options in a new build so user does not have to change again after flashing subsequent times.